modules/apps/frontend-js/frontend-js-spa-web/test/app/App.ts

```diff
@@ -1770,7 +1972,8 @@ describe('App', function () {
 					StubScreen2.prototype.activate,
 					StubScreen.prototype.disposeInternal,
 				];
-				for (var i = 1; i < lifecycleOrder.length - 1; i++) {
+
+				for (let i = 1; i < lifecycleOrder.length - 1; i++) {
 					expect(
 						lifecycleOrder[i - 1].mock.invocationCallOrder[0]
```

Comment:
`Property 'mock' does not exist on type '() => void'`

Not sure what's happening here, I assume that this is another bug that just worked without static types.